banner.html.erb

```diff
@@ -0,0 +1,19 @@
+<style>
```

Comment:
This banner should be removed before merging.

Reply:
Sounds like a good plan.

Reply:
This banner also seems like it's referring to images that don't exist, so I don't think it should stay as-is. I'd be fine if we kept the template here with just an empty `div` or something.